### DISCUSSION TITLE
ProgrammingError at /admin/

### DISCUSSION THREAD

**Discussion by nelsonmandeladev** [cookiecutter/cookiecutter-django#4048] [Q&A]
I'm having this problem while trying to access the Django admin dashboard. Please anyone to help?
![image](https://user-images.githubusercontent.com/69842489/210302385-da06b90f-48a5-45d0-b494-0362fb52e311.png)

**Comment by Afrowave** [comment]
Hi @nelsonmandeladev,

It seems like the database has not been instantiated.

**Comment by nelsonmandeladev** [comment]
Hello @Afrowave I don't think the database instance is the problem because I'm able to create a new instance using post request but I'm just unable to access the Django admin dashboard
